docs/feature_unicode.md

```diff
@@ -83,9 +83,13 @@ The following input modes are available:
 
 ### Switching Input Modes
 
-There are two ways to set the input mode for Unicode: by keycode or by function. Keep in mind that both methods write to persistent storage (EEPROM), and are loaded each time the keyboard starts. So once you've set it the first time, you don't need to set it again unless you want to change it, or you've reset the EEPROM settings.
+The Unicode input mode is stored in two places: in persistent storage (EEPROM) and in memory. The system normally reads the value from EEPROM at init time, and relies on the in-memory value thereafter.
```

Comment:
I'll have to take a look at this again, but .... I'm not happy with the wording here.  By itself ... it's okay. But when taken as a whole, it doesn't flow correctly.